Sources/Core/AST/Stmt/ConditionalCompilationStmt.swift

```diff
@@ -94,10 +94,70 @@ public struct ConditionalCompilationStmt: Stmt {
 
   }
 
+  /// A sequence of structured conditions to elaborate a 1 to N operand compilation condition
+  public indirect enum SequenceCondition: Codable {
+    case operand(Condition)
+    case and(Condition, SequenceCondition)
+    case or(Condition, SequenceCondition)
+
+    public enum ConditionKind: String {
+      case holdOnly
+      case skipMain
+      case skipElse
+    }
+
+    /// Visit the SequenceCondition tail to calculate the right case for executing the right branch
+    private func unroll(for factors: ConditionalCompilationFactors, condCase: ConditionKind) -> Bool
+    {
```

Comment:
```suggestion
    private func unroll(
      for factors: ConditionalCompilationFactors, conditionCase: ConditionKind
    ) -> Bool {
```

We avoid using abbreviations in code and prefer `conditionCase` (which I guess is what you meant here) over `condCase`. The fact that I had to guess is precisely the reason why we stay away from a abbreviations.

Reply:
Looking at use sites, I suspect using simply `condition` would be a better name.